nextclade/defaults/config.yaml

```
@@ -0,0 +1,25 @@
+files:
+    auspice_config: "defaults/auspice_config.json"
+    auspice_json: "auspice/yellow-fever-virus_prM-E.json"
+    clades: "defaults/clades.tsv"
+    colors: "defaults/colors.tsv"
+    include: "defaults/include_strains.txt"
+    reference_prM-E_fasta: "defaults/yellow-fever-virus-reference_prM-E.fasta"
+    reference_prM-E_gff: "defaults/yellow-fever-virus-reference_prM-E.gff"
+strain_id_field: "accession"
+align_and_extract_prM-E:
+    min_length: 650
+    min_seed_cover: 0.01
+filter:
+    group_by: "region year"
+    subsample_max_sequences: 500
+    min_date: 1927
+    min_length: 650
+refine:
+    coalescent: "opt"
+    date_inference: "marginal"
+    clock_filter_iqd: 4
+ancestral:
+    inference: "joint"
+export:
+    metadata_columns: "strain division location region year host"
```

_nit_ 

Noticed in export logs there's a warning that `year` does not exist.  Most of these can also be dropped

```suggestion
    metadata_columns: "strain"
```